cmake/configure.cmake

```diff
@@ -10,6 +10,10 @@ if (NOT CMAKE_BUILD_TYPE)
     set(CMAKE_BUILD_TYPE "Release")
 endif(NOT CMAKE_BUILD_TYPE)
 
+if (MSVC)
+    add_compile_options("/utf-8")
```

Comment:
What is that?

Reply:
Env library can't build without it key with help msvc compiler